src/OpenTelemetry/Internal/OpenTelemetrySdkEventSource.cs

```diff
@@ -162,6 +172,36 @@ public void MetricObserverCallbackError(string metricName, string exception)
             this.WriteEvent(16, metricName, exception);
         }
 
+        [Event(17, Message = "Batcher finished collection with '{0}' metrics.", Level = EventLevel.Informational)]
+        public void BatcherCollectionCompleted(int count)
+        {
+            this.WriteEvent(17, count);
+        }
+
+        [Event(18, Message = "Collection completed in '{0}' msecs.", Level = EventLevel.Informational)]
+        public void CollectionCompleted(long msec)
```

Comment:
nit: seems like an unnecessary conversion of StopWatch to milliseconds. Would ticks work here?